src/status_im2/contexts/chat/messages/content/style.cljs

```diff
@@ -17,7 +18,7 @@
 (defn user-message-content
   [{:keys [first-in-group? outgoing outgoing-status]}]
   {:border-radius      16
-   :padding-horizontal 8
+   :padding-horizontal (if quo.platform/ios? 8 0)
```

Comment:
why is it different for ios ? and `quo` shouldn't be used, there is `react-native.platform`

Reply:
yep definitely, @ajayesivan this might be a good one to start with. We could completely delete quo.platform and update all uses to reference `react-native.platform` 👍 instead

Reply:
thanks my approach to this is not quite correct. I realize this affect the whole list item instead of the selected-item view